lib/rules/color-named/index.js

```diff
@@ -58,11 +58,23 @@ const rule = function (expectation, options) {
         return
       }
 
+      const functionNodes = []
+
       valueParser(decl.value).walk(node => {
         const value = node.value,
           type = node.type,
           sourceIndex = node.sourceIndex
 
+        if (type === "function") {
```

Comment:
Could we get a brief comment stating what this is here for as it's not so obvious and I'm sure we'll come back to it in a couple of months time wondering what it does? :)